docs/contributing/contributing-to-dataline/templates/source-doc-template.md

```diff
@@ -0,0 +1,50 @@
+# Source Name
+
+## Overview 
+
+### Sync Overview
+#### Resulting schema
+Is the schema changed or altered in any way? If so, mention how. 
+
+Does this source have a fixed schema? (e.g: SaaS API like ExchangeRateIO) if so, point to the schema or include it here directly
+
+#### Data Type Mapping
+This section should contain a table mapping each of the source's data types to Dataline types. At the moment, 
+Dataline uses the same types used by Singer.
+
+| Source Type | Dataline Type | Notes
+|-----|-----|-----|
+
+Currently, Dataline uses the same types as Singer: `string`, `int`, `number`, `boolean`, `object`. 
+#### Features
+This section should contain a table with the following format:
+
+| Feature| Supported?(Yes/No) | Notes | 
+|------|-----| ---- |
+| Full Refresh Sync |  | 
```

Comment:
should we just have the first row for now since that's all the platform supports?

Reply:
Those will be coming soon anyways so I'd rather keep them especially since a user will probably want to know if we only support incremental or delete replications etc..